inputs/mvs_config.json

```diff
@@ -375,11 +375,11 @@
             "outflow_direction": "Electricity (DSO)",
             "peak_demand_pricing": {
                 "unit": "currency/kW",
-                "value": 60
+                "value": 0
             },
             "peak_demand_pricing_period": {
                 "unit": "times per year (1,2,3,4,6,12)",
-                "value": 12
+                "value": 1
```

Comment:
This is actually not removing the peak demand pricing (I was alarmed for a second), but disabeling it for the example ;)

Reply:
:D this is what I meant.

If you agree with the changes I would rename the PR before merging as I did not go on with fixing the issue and as the issue is meant to be fixed in #235

Reply:
Fine by me.